events/gameLeave.js

```diff
@@ -0,0 +1,29 @@
+function gameLeave(context, payload) {
+
+    // Get the game's ID
+    const {gameID} = payload;
+
+    // Get the game
+    const game = context.getGameState({gameID});
```

Comment:
For now, lets skip using `gameId`, since if a player is leaving a game they should already be connected to one. Try just calling `context.getGameState()` instead. And to verify, check that `game !== null`.